docs/terms/validation_result.md

```diff
@@ -1,15 +1,91 @@
 ---
 id: validation_result
 title: Validation Result
-hoverText: An object generated when data is Validated against an Expectation Suite.
+hoverText: An object generated when data is Validated against an Expectation or Expectation Suite.
 ---
 import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
 
-<UniversalMap setup='active' connect='active' create='active' validate='active'/>
 
-A Validation Result is an object generated when data is validated against an Expectation Suite.
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='active'/> 
 
+## Overview
 
-NOTES: Everything under here is temporary, for use in writing the detailed page.
-----------
-A report generated from an expectation suite being run against a batch. The validation result is in JSON and is rendered as data docs.
\ No newline at end of file
+### Definition
+
+A Validation Result is an object generated when data is validated against an <TechnicalTag relative="../" tag="expectation" text="Expectation" /> or <TechnicalTag relative="../" tag="expectation_suite" text="Expectation Suite" />.
+
+### Features and promises
+
+Validation Results can be saved for every time that a <TechnicalTag relative="../" tag="checkpoint" text="Checkpoint" /> <TechnicalTag relative="../" tag="validation" text="Validates" /> data, permitting you to maintain records of your data's adherence to Expectations and track trends in the quality of the validated data.  Validation Results are also among the information passed to <TechnicalTag relative="../" tag="action" text="Actions" /> when a Checkpoint is run.  This allows them to be used for any purpose that an Action can be created to handle. 
+
+### Relationship to other objects
+
+Validation Results are generated inside a Checkpoint when a <TechnicalTag relative="../" tag="validator" text="Validator" /> runs an Expectation Suite against its paired <TechnicalTag relative="../" tag="batch_request" text="Batch Request" />.  The Validation Result will be passed to the Checkpoint's `action_list` where it will be available to any Actions that may need it.  The `StoreValidationResultAction` subclass of `ValidationAction` will save Validation Results to the <TechnicalTag relative="../" tag="validation_result_store" text="Validation Result Store" /> if it is present in a Checkpoint's `action_list`.
```

Comment:
Minor suggestion, feel free to ignore.

```suggestion
Validation Results are generated by a Checkpoint when a <TechnicalTag relative="../" tag="validator" text="Validator" /> runs an Expectation Suite against its paired <TechnicalTag relative="../" tag="batch_request" text="Batch Request" />.  The Validation Result will be passed to the Checkpoint's `action_list` where it will be available to any Actions that may need it.  The `StoreValidationResultAction` subclass of `ValidationAction` will save Validation Results to the <TechnicalTag relative="../" tag="validation_result_store" text="Validation Result Store" /> if it is present in a Checkpoint's `action_list`.
```